packages/react/src/app-layout/AppLayout.stories.tsx

```diff
@@ -142,6 +167,108 @@ export const FocusMode: StoryObj<typeof AppLayout> = {
 	args: { focusMode: true },
 };
 
+export const FocusModeWithModal: StoryObj<typeof AppLayout> = {
```

Comment:
I'm not 100% if this story is useful as a design-system story, or if this should be an AG Common thing? @nathsimpson  what do you think?